ini/stm32f4.ini

```diff
@@ -194,6 +194,19 @@ build_flags         = ${stm32_variant.build_flags}
                       -DMF_RX_BUFFER_SIZE=255
                       -DMF_TX_BUFFER_SIZE=255
 
+#
+# Mellow/Fly RRF E3 V1 (STM32F407VGT6 ARM Cortex-M4)
+#
+[env:FLY_RRF_E3_V1]
+extends             = stm32_variant
+board               = marlin_STM32F407VGT6_CCM
+board_build.variant = MARLIN_FLY_RRF_E3_V1
```

Comment:
A custom variant is specified, but there's no `MARLIN_FLY_RRF_E3_V1` folder included. Do you have the variant files or know where to get them?